tox.ini

```diff
@@ -1,34 +1,30 @@
 [tox]
-envlist = py36-flake8, py36-pylint, py36-test, py36-bandit
+envlist = py{36,37-dev,py3},flake8,pylint,test,bandit
 skipsdist=True
+skip_missing_interpreters = {env:TOX_SKIP_MISSING_INTERPRETERS:True}
```

Comment:
By default this will skip missing interpreters, which is nice because if you're like me you don't have python 3.7 or pypy3 installed. On Travis though I configure the repo to have an env variable `TOX_SKIP_MISSING_INTERPRETERS` and it is set to `False`, so on Travis the build will fail if it can't find one of those interpreters.